tasks/ohome-group-write.yml

```diff
@@ -0,0 +1,25 @@
+# Set Group Write Permissions on ORACLE_HOME/dbs for all installed ORACLE_HOMEs
+# Executed only if delphix_user is not equal to "oracle"
+
+- name: Find inventory
+  shell: cat /etc/oraInst.loc | grep inventory_loc | awk -F"=" '{print $2}'
+  register: oraInst
+
+- debug:
+    msg: "{{ oraInst.stdout }}"
+    verbosity: 2
+
+- name: Parse inventory
+  shell: cat "{{ oraInst.stdout }}/ContentsXML/inventory.xml" |grep "HOME NAME" | awk '{print $3}' | awk -F"=" '{print $2}'  | tr -d '"'
```

Comment:
I really like the suggestion, but received error because of a missing dependency:
TASK [ansible-target-host : Parse inventory] **************************************************************************************************************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ImportError: No module named lxml
fatal: [52.39.157.151]: FAILED! => {"changed": false, "msg": "Failed to import the required Python library (lxml) on ip-172-31-62-37.us-west-2.compute.internal's Python /usr/bin/python. Please read module documentation and install in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"}

I'll keep the existing code for now, although I agree it's not as elegant.

Reply:
I also tried doing this as a local delegation, but ran into issues similar to https://github.com/cmprescott/ansible-xml/issues/51 
I'll log this as an RFE on the project:  Use XML parsing instead of shell script to parse inventory